### PR TITLE
starter page

### PR DESCRIPTION
**Pull request by ajackson89**
first pull

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,39 +1,93 @@
...
+    <div class="row">
+      <a class="col S2 waves-effect waves-light btn offset-m5"><i class="material-icons left">search</i>Search</a>
+    </div>
+  </div>
 
+  <!-- Compiled and minified JavaScript for Materialize -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+  <script src="https://code.jquery.com/jquery-3.4.1.min.js"
+    integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
```
Comment (millernj) [inline index.html]: hey, what do we need popper.js for? is that a part of materializes or is it for something else?

Comment (mlfiii) [reply index.html]: Popper.js is required IF you're using Bootstrap JS. It's not required if you're using only the Bootstrap CSS.  So we can remove it.

Comment (millernj) [inline index.html]: Hey, Popperr.js doesn't look like it's needed.

**Review by millernj — CHANGES_REQUESTED** [review]
Can we please take out line 85? Then we should be good to pull

**Review by millernj — APPROVED** [review]

**Pull request merged**